components/DocsLayout.js

```diff
@@ -1,27 +1,65 @@
-import React from 'react'
+import React, { Component } from 'react'
 import Head from './SeoHead'
-import Navbar from './Navbar'
+import Nav from './Nav'
 import { Container, Content, Title } from './Layout'
 
-const DocsLayout = ({ children, title="", description="" }) => (
-  <Container>
 
-    <Head
-      title={`styled-components${title ? `: ${title}` : ''}`}
-      description={description}>
-        <meta name="robots" content="noodp" />
-    </Head>
+class DocsLayout extends Component {
+  state = {
+    isSideFolded: true
+  }
 
-    <Navbar />
+  static defaultProps = {
+    title: '',
+    description: '',
+  }
 
-    <Content>
-      <Title>
-        {title}
-      </Title>
+  onSideFold = () => {
+    let isSideFolded = !this.state.isSideFolded
+    // if(!isSideFolded) {
```

Comment:
Can you remove the commented out code here? If you need it back we'll have the git history anyway :wink:

Reply:
Sorry! It was my pre-git habit. You won't see this kind of commenting later on.

Reply:
No worries! I sometimes forget this stuff too :)